user-manual/access-content/context-menu.rst

```diff
@@ -437,7 +437,7 @@ elements:
 Subject and Place browse pages
 ==============================
 
-:term:`Subject` and :term:`Place` terms can be used in AtoM as
+:term:`Subject` and :term:`Places` terms can be used in AtoM as
```

Comment:
Hi @peterVG  - couple things about this change. First, all terms in the glossary are in singular format, so this change will actually cause an error if you run the `make` command. Did you do that and check for errors first, before pushing? 

In any case, I'm not sure I agree with this particular change - I noticed that you didn't change Subject to "Subjects" for example. It's the Subjects and Places taxonomy, but I think it's okay to individually call them Subject and Place terms. I can still change the label and make the glossary link work if you think this is important, like so:

```
:term:`Places <place>`
```

However, again I think we should consider if it's important, and correct.  

More importantly however - did you "Artefactualize" your local env, so that it is tracking the internal gitolite repository, before pushing this change? If not, then this change won't stick when merged! I'll make whatever changes are needed locally since I'm all set up :)

SAD team has it as an upcoming task to finally get our docs (and AtoM itself) off gitolite, but that hasn't happened yet. I have some internal docs on how to Artefactualize the vagrant box, if needed, though things will be a bit different if running everything locally.

Reply:
Super sorry about that. I saw your change from Place to Places in a commit somewhere else and thought you were trying to standardize on that. It was my first comment in the review and realized right after that pushing commits really wasn't the best way to do that. I switched to inline comments after that. I didn't realize that this would also affect the glossary and no, I didn't run `make`. Hopefully you can just ignore/remove the commit from your PR?
Lastly, I'm not sure what is involved in "Artefactualizing" my local env. Please send me the link. FYI I cloned the GH repo to build my Sphinx docs but pushed this commit via the GH UI. I guess in either case that's not correct as we're working off a gitolite repo as master?